.github/workflows/build-test-core-win64.yml

```diff
@@ -0,0 +1,81 @@
+# AUTOGENERATED FROM build-test-core-win64.jsonnet DO NOT MODIFY
+name: build-test-core-win64
+"on":
+  workflow_dispatch:
+  workflow_call:
+jobs:
+  job:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: bash
```

Comment:
maybe add some comments on why we need those new defaults here; why bash?
and why we need a working-directory that is different?

Reply:
removed the working-directory stuff (worked around with the `opam-local-packages: dont_install_local_packages.opam` line)